.clang-format

```diff
@@ -13,3 +13,7 @@ UseTab: Never
 IndentWidth: 4
 BreakBeforeBraces: Linux
 AccessModifierOffset: -4
+ForEachMacros:
+  - 'for_each_set_bit'
+  - 'for_each_set_bit_from'
+  
```

Comment:
No! My intention was to ask you to configure your local Git line encoding properly. Check this: https://help.github.com/en/github/using-git/configuring-git-to-handle-line-endings
Don't add a new line.

Reply:
If you browse via GitHub web, you will figure out the red sign in the end of `.clang-format`.

Reply:
I follow the guide but it still has "No new line at end of file".
I'm trying to figure it out.

Reply:
I don't know why I can't solve that problem with that guide.
But I find that ""No new line at end of file" is because I use VS code, So I use vim to solve it.